scripts/check-prebuild.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+/* eslint-disable import/no-commonjs */
+
+const fs = require('fs');
+const path = require('path');
+
+const prebuildFolders = ['ios', 'android'];
+
+const notExist = prebuildFolders.some((folder) => {
+  const folderPath = path.join(__dirname, '../example', folder);
+  return !fs.existsSync(folderPath);
+});
+
+if (notExist) {
+  const message = `Native source code folders ${prebuildFolders.join(
```

Comment:
Maybe instead of throwing we can just run `expo prebuild`?

Reply:
Fixed in [145d4a3](https://github.com/react-navigation/react-navigation/pull/11020/commits/145d4a39101e0d04320edde9b105279a333fae6f)